frontend/src/lib/components/EntityFilterInfo.tsx

```diff
@@ -11,11 +10,21 @@ interface EntityFilterInfoProps {
     style?: React.CSSProperties
 }
 
-function TextWrapper(props: TextProps): JSX.Element {
+function TextWrapper({
+    title,
+    ellipsis = false,
+    className = '',
+    children,
+}: {
+    title: string | undefined
+    ellipsis?: boolean
+    className?: string
+    children: React.ReactNode
+}): JSX.Element {
     return (
-        <Typography.Text style={{ maxWidth: 400 }} {...props}>
-            {props.children}
-        </Typography.Text>
+        <div className={clsx('EntityFilterInfo', ellipsis && 'truncate', className)} title={title}>
```

Comment:
Usually the top-level element's class in our usage of BEM is the component name, so `TextWrapper` here.
Though TBH I think we can just completely do away with `TextWrapper`, it's styling looks like unnecessary complexity, especially if `showSingleName` is false

Reply:
Seconded. We can likely get rid of `TextWrapper` completely, or then at least move the styles from the separate scss file into the component as tailwind classes.